libsplinter/src/admin/store/mod.rs

```diff
@@ -336,10 +321,10 @@ impl CircuitPredicate {
     }
 }
 
-/// Defines methods for CRUD operations and fetching and listing circuits, proposals, nodes and
-/// services without defining a storage strategy
+/// Interface for performing CRUD operations on circuits, proposals, nodes, and services that does
```

Comment:
Not really sure that the "that does not define a storage strategy" part of the comment is necssary, as this is what traits impl by definition.

Reply:
Removed that bit (also in the mod doc)